graphrag_sdk/agents/kg_agent.py

```
@@ -1,5 +1,6 @@
-from graphrag_sdk.kg import KnowledgeGraph
 from .agent import Agent
+from typing import Optional
```

Ruff

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)